mars/services/scheduling/worker/workerslot.py

```diff
@@ -269,13 +270,16 @@ async def upload_slot_usages(self, periodical: bool = False):
             if global_slots_ref is not None:  # pragma: no branch
                 # FIXME fix band slot mistake
                 delays.append(
-                    global_slots_ref.update_subtask_slots.delay(
-                        self._band[1], session_id, subtask_id, max(1.0, usage)
+                    global_slots_ref.update_subtask_resources.delay(
+                        self._band[1],
+                        session_id,
+                        subtask_id,
+                        Resource(num_cpus=max(1.0, usage)),
```

Comment:
This line also have GPU issue.

Reply:
Is there an API to check current band is a gpu band in `BandSlotManagerActor`?